packages/create-cloudflare/templates/hono/snippets/bindingsType.ts

```diff
@@ -1 +1,3 @@
-type Bindings = Record<string, unknown> & CloudflareBindings
+type Bindings = {
+  [key in keyof CloudflareBindings]: CloudflareBindings[key]
+}
```

Comment:
```suggestion
}

```

Reply:
I've really gotta setup some tooling to fix this 🤦